src/components/drawing/index.js

```diff
@@ -112,6 +112,7 @@ drawing.lineGroupStyle = function(s, lw, lc, ld) {
 };
 
 drawing.dashLine = function(s, dash, lineWidth) {
+    lineWidth = +lineWidth || 0;
```

Comment:
Maybe - I'll try that tonight and see if it implies any further fixes. Any other IE issues I should include in my testing?